music21/midi/translate.py

```diff
@@ -3711,7 +3711,6 @@ def procCompare(mf_inner, match_inner):
                  (0, 'SET_TEMPO', None),
                  (1024, 'END_OF_TRACK', None),
                  (0, 'SEQUENCE_TRACK_NAME', None),  # Music track
-                 (0, 'PROGRAM_CHANGE', None),
```

Comment:
This looked like an improvement (same message duplicated after the pitch bend message). Must be doing a better job of deduplicating generic instruments?